### ISSUE TITLE
Hyperlink to a URL

### ISSUE THREAD

**Issue by EricCBonet** [JelteF/PyLaTeX#264]
I would like to add a text with a hyperlink to my document being generated via pyLaTeX. In LaTeX this would be performed by using:

```\href{http://www.sharelatex.com}{Something Linky}```

I have found the [labelref documentation](https://jeltef.github.io/PyLaTeX/current/pylatex/pylatex.labelref.html) containing the Marker object as well as the Hyperref Object but i cannot seem to get it working. 

```
from pylatex import Document, Hyperref

doc = Document()
doc.append("Example text a link should go ")
# i was hoping the hyperlink would work like i'm showing below
doc.append(Hyperref("https://jeltef.github.io/PyLaTeX", "here"))
doc.generate_pdf("Example_document", clean_tex=True)
```

Running the following code produces the a pdf document without errors. [image of the document produced][1]

While what i would expect is that the word "here" is a hyperlink and shown in blue.

Is this possible?

  [1]: https://i.stack.imgur.com/Kom35.png

**Comment by jeffheaton** [comment]
I do not believe that is supported, see here:  https://stackoverflow.com/questions/57059457/python-pylatex-hyperlink-to-a-url

This is the method that I used to support it.  First, define a hyperlink class:

```
def hyperlink(url,text):
    text = escape_latex(text)
    return NoEscape(r'\href{' + url + '}{' + text + '}')
```

Then I just do this:

`document.append(hyperlink("http://www.jeffheaton.com", "My Site"))`
